docs/platforms/apple/common/configuration/app-hangs.mdx

```diff
@@ -184,3 +184,40 @@ SentrySDK.start { options in
     options.enableReportNonFullyBlockingAppHangs = NO;
 }];
 ```
+
+## App Hang Rate
+
+App hangs occur when the main UI thread of an application becomes unresponsive. When configured, Sentry's Apple SDK reports app hang events, and Sentry calculates an App Hang Rate based on these events and user sessions.
+
+App Hang Rate is the percentage of unique users who have experienced a **Fatal App Hang** during the specified time period.
+
+<Alert level="info">
+
+The App Hang Rate in Sentry only accounts for **Fatal App Hangs** - app hangs that lead to your app being terminated, either by the user or the OS watchdog. Non-fatal app hangs are captured as events but do not contribute to the App Hang Rate calculation.
+
+</Alert>
+
+### Fatal App Hangs
+
+Fatal app hangs are the most critical type of app hang because they result in app termination and directly impact user experience. These occur when:
+
+- The user manually force-quits the app during a hang
+- The system watchdog terminates the app due to unresponsiveness
+
+The SDK automatically detects and reports fatal app hangs starting with version 8.50.0. Fatal app hangs are distinguished by their `exception.type`:
```

Comment:
`h`: We must mention here that users need to enable `options.enableAppHangTrackingV2 = true`. We must also mention this earlier in this App Hang Rate section, cause otherwise it will confuse users.